NOTICE.txt

```diff
@@ -113,6 +148,19 @@ SOFTWARE.
 
 ---
 
+## @sentry/react-native
```

Comment:
Was added via https://github.com/mattermost/mattermost-mobile/commit/1efb01deda81cc0ee0fa0a36af8c02e342272f11#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

Reply:
@enahum If you can provide a link for this and then I can fill in the details.